account-kit/rn-signer/src/client.ts

```diff
@@ -244,52 +284,162 @@ export class RNSignerClient extends BaseSignerClient<undefined> {
   /**
    * Retrieves the list of MFA factors configured for the current user.
    *
-   * @throws {Error} This method is not implemented in RNSignerClient
+   * @returns {Promise<{ multiFactors: MfaFactor[] }>} A promise that resolves to an array of configured MFA factors
+   * @throws {Error} If no user is authenticated
    */
-  public override getMfaFactors(): Promise<{ multiFactors: MfaFactor[] }> {
-    throw new Error("getMfaFactors is not implemented in RNSignerClient");
-  }
+  public override getMfaFactors = async (): Promise<{
```

Comment:
shouldn't the base implementation just work here?

Reply:
i had this question while working on my "api key signer" client impl as well... the MFA methods are all abstract on the `BaseSignerClient`, unlike many of the other methods (whoami, signRawMessage, etc.)

Reply:
oh yea I just went to check that too. that seems unexpected. there's nothing in these definitions that's environment specific from what I can see

Reply:
Yeah that is a good point, I didn't think about that.  When we first got started with MFA I was only implementing it for the web.  I don't think it ever clicked for me that all these methods can just live in the base client.

I can update that as part of this PR

Reply:
sounds good, thanks!